pkg/functions/function.go

```diff
@@ -96,6 +100,10 @@ type BuildSpec struct {
 
 	// Build Env variables to be set
 	BuildEnvs []Env `yaml:"buildEnvs"`
+
+	// PVCSize specifies the size of persistent volume claim used to store function
+	// when using deployment and remote build process (only relevant when Remote is true).
+	PVCSize string `yaml:"pvcSize"`
```

Comment:
Can you add `omitempty` here?

Reply:
Actually not.

Reply:
Well I am not entirely sure now I think about it. We always default it right @grafvonb , so it is never empty.